Makefile

```diff
@@ -56,3 +60,16 @@ sync: install
 
 update-blocklist-file:
 	wget -O blocklist.xml $(BLOCKLIST_FILE_URL)
+
+update-schemas:
+	wget -O schemas.json $(AMO_BLOCKLIST_UI_SCHEMA)
+
+install-kinto: $(VENV)/bin/kinto
+$(VENV)/bin/kinto: install
+	$(VENV)/bin/pip install kinto
+
+run-kinto: $(VENV)/bin/kinto
+	$(VENV)/bin/kinto --ini config/kinto.ini start
+
+need-kinto-running:
+	@curl http://localhost:8888/v1 2>/dev/null 1>&2 || (echo "Run 'make run-signer' before starting tests." && exit 1)
```

Comment:
So basically this line is making sure kinto is running before starting the functional tests. ([See here](https://github.com/mozilla-services/xml2kinto/pull/34/files#diff-b67911656ef5d18c4ae36cb6741b7965R45))